python/py-rfc6555/Portfile

```
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-rfc6555
+version             0.0.0
+revision            0
+categories-append   devel
+platforms           darwin
+license             Apache-2
+supported_archs     noarch
+
+maintainers         nomaintainer
+
+description         Python implementation of RFC 6555.
+long_description    Python implementation of the Happy Eyeballs Algorithm described in RFC 6555. \
+    Provided with a single file and dead-simple API \
+    to allow easy vendoring and integration into other projects.
+
+homepage            https://www.github.com/SethMichaelLarson/rfc6555
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  7bd0464b9353f10892fca2cca5943962ec5f9cfa \
+                    sha256  191cbba0315b53654155321e56a93466f42cd0a474b4f341df4d03264dcb5217 \
+                    size    11218
+
+python.versions     27 35 36 37 38 39
```

```suggestion
python.versions     38 39
```

I'd stick with 38 and 39 for now. New versions can then be added later.

See https://github.com/macports/macports-ports/pull/8620#issuecomment-703742167 for more info.

---

@harens I created this port for https://github.com/macports/macports-ports/pull/9352 and this one is stuck on 27.

I can keep `27 38 39` but it is a quite wired.

---

Thanks @catap for your response. I only noticed https://github.com/macports/macports-ports/pull/9352 after I sent the review, so I edited my changes accordingly at the time.

As for `27 38 39`, other ports have only these python versions (e.g. [py-scgi](https://github.com/macports/macports-ports/blob/6661ca6017ff18269625a52a0cda17b49829e5f2/python/py-scgi/Portfile), [py-dulwich](https://github.com/macports/macports-ports/blob/578497d8c50cbf3b92c6c50084a9e738ee92e454/python/py-dulwich/Portfile), etc.), so I think it should be suitable here as well.